main/webapp/WEB-INF/velocity.properties

```diff
@@ -5,7 +5,4 @@ parser.pool.size=5
 
 velocimacro.context.localscope=false
 velocimacro.library.autoreload=false
-velocimacro.permissions.allow.inline.local.scope=true
-
-input.encoding=UTF-8
```

Comment:
wouldn't our Velocity templates (or anyone's) always typically be encoded as UTF-8 as a best practice?